tests/foreman/api/test_multiple_paths.py

```diff
@@ -502,19 +489,18 @@ def test_delete_and_get(self):
 
 
 class EntityReadTestCase(APITestCase):
-    """
-    Check that classes inheriting from
+    """Check that classes inheriting from
```

Comment:
`Check that classes are inherited from` or `Check that classes are being inherited from`